homework_6/задания 3 уровня/3-валидация_формы/validation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        input {
+            border-style: solid;
+        }
+    </style>
+</head>
+
+<body>
+    <form>
+        <input id="first" type="text"><br>
+        <br>
+        <input id="second" type="text"><br>
+        <br>
+        <button>Отправить</button>
+    </form>
+
+    <!-- 
+        Задание. Вам нужно проверять пусто ли в input'ах при отправке формы.
+        Если в инпуте пусто делаете ему красную рамку. Если хотя бы в одном
+        инпуте пусто форма не должна отправляться.
+        1. сохраните в переменную formEl ссылку на форму
+        2. сохраните в переменную ссылку на input#first
+        3. сохраните в переменную ссылку на input#second
+        4. на formEl назначьте обработку события отправки формы (submit),
+        у функции-обработчика объявите параметр event.
+            4.1 в функции-обработчике вы проверяете пусты ли input'ы, для этого
+            читайте у них свойство value
+            4.2 если input пустой тогда вы обращаетесь к его style.borderColor
+            и делаете его равным red
+            4.3 чтобы остановить отправку формы используйте event.preventDefault()
+     -->
+
+    <script>
+        let formEl = document.querySelector("form");
+        let inputList = document.querySelectorAll("input");
+
+        formEl.addEventListener("submit", (event) => {
+            inputList.forEach((input) => {
+                if (input.value == "") {
+                    input.style.borderColor = "red";
+                    event.preventDefault()
+                }
```

Comment:
В задании это не сказано, но думаю стоит сразу иметь ввиду, что этот код визуально навсегда делает инпут красным, даже когда в него введут текст, он останется красным, можно минимальным кодом добавить ему шанс на возвращение в валидный вид
```suggestion
                if (input.value == "") {
                    input.style.borderColor = "red";
                    event.preventDefault()
                }else{
                    input.style.borderColor = "";
                }
```